example/boxed.txt

```diff
@@ -14,19 +14,19 @@ to run each test in a controlled subprocess.  Here is a basic example::
     @pytest.mark.parametrize("arg", range(50))
     def test_func(arg):
         time.sleep(0.05) # each tests takes a while
-        if arg % 19 == 0: 
+        if arg % 19 == 0:
```

Comment:
I think we should add to the top of this file:

```
.. note::

  Since 2.0, the actual implementation of the ``--boxed`` option has been moved to a 
  separate plugin, `pytest-forked <https://github.com/pytest-dev/pytest-forked>`_ 
  which can be installed independently. The ``--boxed`` command-line options remains 
  for backward compatibility reasons.
```